src/property-filter/styles.scss

```diff
@@ -56,10 +56,18 @@
     flex-shrink: 0;
     margin-left: awsui.$space-xl;
   }
+  &-form {
+    margin-bottom: awsui.$space-scaled-l;
+  }
   &-actions {
     display: flex;
     justify-content: flex-end;
-    margin-top: awsui.$space-scaled-xl;
+    padding-top: awsui.$space-s;
```

Comment:
Should it be a `space-scaled-s` token to match the former margin-top?

Reply:
I took the date range picker as an example. There we have `space-scaled-l` to the top and `space-s` to the bottom of the separator.

Reply:
Fine, works for me